client/environment_test.go

```diff
@@ -11,11 +11,11 @@ import (
 func TestCreateEnvironment(t *testing.T) {
 	req := models.CreateEnvironmentRequest{
 		EnvironmentName:  "env_name",
+		EnvironmentType:  models.EnvironmentTypeStatic,
```

Comment:
Should we test dynamic as well here?

Reply:
I'm not convinced that would add much value; I'd be more open to something like adding tests for the models' `Validate()` functions.

Reply:
Okay I'm just wondering if it should be tested somewhere. So that would work.